### DISCUSSION TITLE
Should ChimeraCSS and ChimeraDemo repos be moved to the chimera org?

### DISCUSSION THREAD

**Discussion by MarkusJohansen** [ChimeraCSS/ChimeraCSS#113] [Q&A] [answered]
I think it would be healthier for the framework to move the belonging repos to a currently unused GitHub org, for easier maintenance and adding more people as admins/maintainers (currently i have @SindreKjelsrud in mind). What is your take on this?

**Comment by SindreKjelsrud** [comment, marked as answer]
I think your reasoning for it is good! Fully support it :100:
